configuration/mounting-paritions/en.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Automatically mounting internal partitions"
++++
+# Automatically mounting internal partitions
+
+Solus can automatically mount your internal partitions on startup.
+
+## Budgie & GNOME
+
+First, open the application Disks. Select the disk from the left you want to automatically mount. 
```

Comment:
All of these steps should using numbered lists. You should also accompany this with images.